### DISCUSSION TITLE
view all data in the table without pagination

### DISCUSSION THREAD

**Discussion by aynuayex** [TanStack/table#5465] [Q&A]
hi, everyone i was using [datatable](https://ui.shadcn.com/docs/components/data-table) component from shadcn-ui library and it uses tanstack table to build it and this library shows only 10 rows data even if there are more than 10 data, it only shows  all the data if there is pagination elements.

so, my question is; is there any other way or method i can show all the data without using pagination?

**Comment by krishna-as-dev** [comment]
it might be late but I found this solution that works for me. pass `manualPagination` as true and that’s it. due to the manualPagination it showcase all the data that we pass and thus showing all the data together then you can add scroll to the table.
to hide the pagination buttons in the end you can make a custom prop like `hidePaginationOptions`
